pymc/sampling/jax.py

```diff
@@ -142,6 +143,22 @@ def _sample_stats_to_xarray(posterior):
     return data
 
 
+def _postprocess_samples(
+    jax_fn: List[TensorVariable],
+    raw_mcmc_samples: List[TensorVariable],
+    postprocessing_backend: str,
+    num_chunks: int = 1,
```

Comment:
(1) I think we should add a branch here to allow user to use `vmap(vmap(...))`, maybe even with `num_check = None`.
(2) Is `num_chunks = 1` a good default? I have no experience but I see in your snippet on discourse you are using `10`.

Reply:
I can make the argument `Optional` with `None` as the default and use `vmap` in this case?

I think it would be cleaner to only introduce one extra argument, because vmap with a chunksize argument wouldn't make sense

Reply:
I chose 1 as the default argument because its behaviour should be very similar to vmap.